source/publishing_data/12_translating_dataset/translating_dataset.rst

```diff
@@ -0,0 +1,88 @@
+Translating a dataset
+=====================
+
+For domains with more than one language activated in the front office, Opendatasoft allows the translation of specific elements of the datasets, as well as their metadata. It should help users understand a dataset as best as possible, even though the data itself is not in their preferred language.
+
+In the Opendatasoft platform, these translations are done in the Translations tab of the edition interface of each dataset.
+
+.. image:: images/translation_tab_interface.png
+
+Choosing elements for translation
+---------------------------------
+
+For each available language displayed in the Translations tab, it is possible to translate various elements that have been originally configured in 2 other configuration tabs of the dataset:
+
+- Information, which 4 following metadata can be translated: Title, Description, Keywords and Publisher.
+- Processing, which fields names and descriptions can be translated.
+
+.. admonition:: Note
+   :class: note
+
+   For each language, for both Information and Processing, a progress bar is displayed to indicate how much of the elements have already been translated.
+
+To start translating chosen elements from the dataset:
+
+1. Choose an available language.
+2. For the chosen language, choose whether to translate elements from the Information or the Processing tab.
+3. For the chosen tab, click on the related Edit link. A translation window opens.
+
+.. image:: images/translation_window.png
+
+.. admonition:: Important
+   :class: important
+
+   The translation window only enables the translation of elements that have already been configured in the original language. For example, if the Publisher metadata has not been configured in the Information tab, it will not be possible to translate it.
```

Comment:
We could explain that we translate term and not element: if the publisher changes, the translation is lost. @richterb Am I right ?

Reply:
Yes I mentioned in my review that this would interesting to mention. And this is probably the right place to mention it.